README.rst

```diff
@@ -1,2 +1,54 @@
 This package provides an ExplainCollection class
```

Comment:
Let's quote the names of all classes - ``ExplainCollection``

Reply:
Done.